### DISCUSSION TITLE
Trouble with basic setup code for chromadb - generates numpy type error

### DISCUSSION THREAD

**Discussion by robot1125** [vanna-ai/vanna#501] [Q&A] [answered]
Hey all, I'm trying to run the basic setup code from the docs:

```
from vanna.ollama import Ollama
from vanna.chromadb import ChromaDB_VectorStore

class MyVanna(ChromaDB_VectorStore, Ollama):
    def __init__(self, config=None):
        ChromaDB_VectorStore.__init__(self, config=config)
        Ollama.__init__(self, config=config)

vn = MyVanna(config={"model": "llama3:22B"})
```

I'm using python 3.11.6, vanna 0.6.0, chromadb 0.5.0, and numpy 2.0.0.

Running the code above generates this error on the vanna.chromadb import line:

```
AttributeError: `np.float_` was removed in the NumPy 2.0 release. Use `np.float64` instead.. Did you mean: 'float16'?
```

Can anyone point me at a solution?

**Comment by zainhoda** [comment, marked as answer]
It looks like an underlying issue with Chroma:

https://github.com/chroma-core/chroma/issues/2358

For now, you can try this:

```bash
pip uninstall numpy
pip install 'numpy<2.0.0'
```

Either chroma has to become compatible with numpy 2.0 or they need to add the specific version to their requirements.

**Reply by robot1125** [reply]
Thank you, that's the answer.
